### PR TITLE
Create users and stories

### PR DESCRIPTION
**Pull request by kenyonj**

### CODE REVIEW

File: spec/factories/stories.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Read about factories at https://github.com/thoughtbot/factory_girl
+
+FactoryGirl.define do
```
Comment (jsteiner) [inline spec/factories/stories.rb]: Just add this to `spec/factories.rb` and get rid of this dir.

File: spec/models/story_spec.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 require 'spec_helper'
 
-describe User do
+describe Story do
```
Comment (jsteiner) [inline spec/models/story_spec.rb]: You can remove this if it ain't doing anything.

**Comment by jsteiner** [comment]
couple comments. otherwise :thumbsup:

File: app/views/application/_errors.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<% if target.errors.any? %>
+<div id="error_explanation">
```
Comment (gabebw) [inline app/views/application/_errors.html.erb]: indent

File: db/migrate/20140513170328_create_stories.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class CreateStories < ActiveRecord::Migration
+  def change
+    create_table :stories do |t|
+      t.belongs_to :user, index: true
```
Comment (gabebw) [inline db/migrate/20140513170328_create_stories.rb:4]: I'd make this `null: false`

**Pull request closed without merging**